ExampleApp/ExampleApp/AudioPlayerView.m

```diff
@@ -254,6 +271,27 @@ -(void) muteButtonPressed
 	}
 }
 
+-(void) rateControlChanged
+{
+    switch (rateControl.selectedSegmentIndex) {
```

Comment:
Sorry @tumtumtum. This was an unintended PR with many improper changes and inconsistent coding style. Will reopen once the changes are tested more and made consistent.